conanfile.py

```diff
@@ -8,51 +8,47 @@
 class FlacConan(ConanFile):
     name = "flac"
     version = "1.3.2"
-    description = "Free Lossless Audio Codec "
-    url = "https://github.com/xiph/flac"
+    description = "Free Lossless Audio Codec"
+    homepage = "https://github.com/xiph/flac"
+    url = "https://github.com/bincrafters/conan-flac"
     license = "BSD"
     exports = ["LICENSE.md"]
     exports_sources = ["CMakeLists.txt"]
     generators = "cmake"
     source_subfolder = "sources"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "use_asm": [True, False]}
-    default_options = "shared=False", "use_asm=False"
+    options = {"shared": [True, False], "use_asm": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "use_asm=False", "fPIC=True"
+    requires = "ogg/1.3.3@bincrafters/stable"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
 
     def build_requirements(self):
         if self.options.use_asm:
             self.build_requires("nasm/2.13.01@conan/stable")
 
-    def requirements(self):
-        self.requires("ogg/1.3.3@bincrafters/stable")
-
     def source(self):
-        source_url = "http://downloads.xiph.org/releases/flac/"
-        tools.download("{0}/flac-{1}.tar.xz".format(source_url, self.version),
-                       "flac.tar.xz")
-        self.run("cmake -E tar xf flac.tar.xz")
+        tools.get("{0}/archive/{1}.tar.gz".format(self.homepage, self.version))
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self.source_subfolder)
 
     def build(self):
         cmake = CMake(self)
         cmake.definitions["CONAN_ARCH"] = str(self.settings.arch)
-        cmake.definitions["USE_ASM"] = "ON" if self.options.use_asm else "OFF"
+        cmake.definitions["USE_ASM"] = self.options.use_asm
+
+        if self.settings.os != "Windows":
+            cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.fPIC
+
         cmake.configure()
         cmake.build()
+        cmake.install()
 
     def package(self):
-        include_dir = os.path.join(self.source_subfolder, "include")
-        
-        self.copy(pattern="LICENSE")
-        self.copy(pattern="*.h", dst="include", src=include_dir)
-        self.copy(pattern="*.hh", dst="include", src=include_dir)
-        self.copy(pattern="*.hpp", dst="include", src=include_dir)
-        self.copy(pattern="*.dll", dst="bin", keep_path=False)
-        self.copy(pattern="*.lib", dst="lib", keep_path=False)
-        self.copy(pattern="*.a", dst="lib", keep_path=False)
-        self.copy(pattern="*.so*", dst="lib", keep_path=False)
-        self.copy(pattern="*.dylib", dst="lib", keep_path=False)
+        self.copy(pattern="LICENSE.md", dst="licenses", keep_path=False)
```

Comment:
Note: we don't copy the recipe license (LICENSE.md) to the package directory. Only the library license.

Reply:
Ok fixing it